waku/v2/protocol/waku_store/protocol.nim

```diff
@@ -147,8 +146,10 @@ proc findMessages(w: WakuStore, query: HistoryQuery): HistoryResponse {.gcsafe.}
     error: HistoryResponseError.NONE
   )
 
-proc init*(ws: WakuStore, capacity = StoreDefaultCapacity) =
-
+proc initProtocolHandler*(ws: WakuStore) =
+  # TODO: If this line is commented/removed, the code does not compile! WTF!?
```

Comment:
🤔

Reply:
Sorry... My astonishment notes leaked 🤦🏼 

But it is true:

![image](https://user-images.githubusercontent.com/3949095/191061353-145a397b-428e-4030-b03b-1d6b20713c1f.png)

And the import is in place:

```nim
  ../../utils/pagination,
```

Reply:
I have solved it by re-exporting the `pagination` module in the `waku_store_queue` module. These are the kind of problems that I don't like 😓 

That's why I work towards a good and easy to follow module graph 🔀